examples/gno.land/p/nt/simpledao/dao.gno

```diff
@@ -0,0 +1,206 @@
+package simpledao
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/gov/dao"
+	pproposal "gno.land/p/gov/proposal"
+)
+
+var (
+	errInvalidExecutor           = errors.New("invalid executor provided")
+	errInsufficientProposalFunds = errors.New("insufficient funds for proposal")
+	errInsufficientExecuteFunds  = errors.New("insufficient funds for executing proposal")
+	errProposalExecuted          = errors.New("proposal already executed")
+	errProposalInactive          = errors.New("proposal is inactive")
+	errProposalExpired           = errors.New("proposal is expired")
+	errProposalNotAccepted       = errors.New("proposal is not accepted")
+
+	errNotGovDAO = errors.New("caller not correct govdao instance")
+)
+
+var (
+	minProposalFeeValue int64 = 100 * 1_000_000 // minimum gnot required for a govdao proposal (100 GNOT)
+	minExecuteFeeValue  int64 = 500 * 1_000_000 // minimum gnot required for a govdao proposal (500 GNOT)
+
+	minProposalFee = std.NewCoin("ugnot", minProposalFeeValue)
+	minExecuteFee  = std.NewCoin("ugnot", minExecuteFeeValue)
+)
+
+// SimpleDAO is a simple DAO implementation
+type SimpleDAO struct {
+	membStore *MembStore
+	propStore *PropStore
+}
+
+// NewSimpleDAO creates a new instance of the simpledao DAO
+func NewSimpleDAO(membStore *MembStore, propStore *PropStore) *SimpleDAO {
```

Comment:
```suggestion
func New() *SimpleDAO {
```

Reply:
Are you proposing just the rename, or dropping the params as well?

Reply:
I'm assuming both, since you suggested flattening the API

Reply:
Rename since there is a single object to “new”, which is the actual super star of this package :)

and yes, no argument needed.